http/filestream/filestream.go

```diff
@@ -42,11 +42,17 @@ func readFile(fileReader *multipart.Part, fileWriterFunc FileWriterFunc) (err er
 	if err != nil {
 		return err
 	}
-	defer ioutils.Close(fileWriter, &err)
-	if _, err = io.Copy(fileWriter, fileReader); err != nil {
+	var writers []io.Writer
+	for _, writer := range fileWriter {
+		defer ioutils.Close(writer, &err)
+		// Create a multi writer that will write the file to all the provided writers
+		// We read multipart once and write to multiple writers, so we can't use the same multipart writer multiple times
+		writers = append(writers, writer)
```

Comment:
I think this defer can be problematic inside a for loop. Maybe it requires another defered loop of writers closure.

Reply:
What is the concern you are referring to?

Reply:
You can ignore this comment. After reviewing it again, I don't think it will be problematic in this case in the end
@Or-Geva